modules/ipc/include/hephaestus/ipc/zenoh/action_server/types.h

```diff
@@ -5,9 +5,17 @@
 #pragma once
 
 #include <cstdint>
+#include <string>
 
 namespace heph::ipc::zenoh::action_server {
 
+template <typename RequestT>
+struct Request {
+  RequestT request;
+  std::string response_service_topic_name;
```

Comment:
wouldn't it be sufficient to just pass in the base topic name and the uid and derive the two topic on top of that instead of sending two strings?